pkg/apis/routing/v1alpha1/routing_strategy_multiarmbandit.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2020 kubeflow.org.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+//
+type MultiArmBanditSpec struct {
+	EpsilonGreedy *EpsilonGreedySpec `json:"epsilonGreedy,omitempty"`
+}
+
+type EpsilonGreedySpec struct {
```

Comment:
Not confident about this yet, consider leaving this out. Would love to see a proposal and how this evolves.